### ISSUE TITLE
Feedback on Milestone 3 report

### ISSUE THREAD

**Issue by pooyanjamshidi** [csce585-mlsystems/CVDisasterRescue#7]
Thanks for submitting the milestone-3 report!

It was nice seeing the results, and I enjoyed reading your report; it is well-written with enough details, though there is room for improvement:

1. Please add details about each individual model and training and testing, in particular, what dataset was used to train each model and the training process, including any algorithm and hyperparameters. 
2. Figures 2 and 3 are unclear; the y-axis mentions the number of trials, while the title refers to accuracy. Overall, I would suggest adding a bit of description in the caption to clarify the results. 
3. Have you tried to fine-tune the sound classification model by adding the data you collect by your mic array to the training set and see whether you get better accuracy? 
4. Please ensure your final report contains all materials, including related work, and everything requested in the project tab of the course website. 

Finally, thanks for your efforts during the semester; I could see you are motivated enough, and I hope this course and the project were helpful for you.
